lte/gateway/python/magma/mobilityd/rpc_servicer.py

```diff
@@ -64,6 +64,10 @@ def __init__(self, mconfig, config):
         ip_block = _get_ip_block(mconfig.ip_block)
         if ip_block is not None:
             try:
+                #forcefully remove the existing ip blocks
+                self._ipv4_allocator.remove_ip_blocks(
```

Comment:
If we're gonna add this set behavior for this servicer, would it make more sense to either update the name to `SetIPBlock` or either add another separate call? As of now it's confusing to say we're calling to add an IP block and then removing all of the previous allocated ones. Let's chat more on this

Reply:
I think the semantics of add and remove are clear in the RPC service. Do we use these RPC calls beyond CLI and S1AP testing?

What really matters in the operational set up is what the cloud streams down as the configuration and this initialization step seems to be the only place to pick up that configuration and set ip block.